printer/html.go

```diff
@@ -63,7 +64,7 @@ func (p *html) PrintClones(dups [][]*syntax.Node) error {
 
 	sort.Sort(byNameAndLine(clones))
 	for _, cl := range clones {
-		fmt.Fprintf(p.w, "<h2>%s:%d</h2>\n<pre>%s</pre>\n", cl.filename, cl.lineStart, cl.fragment)
+		fmt.Fprintf(p.w, "<h2>%s:%d</h2>\n<pre>%s</pre>\n", cl.filename, cl.lineStart, htmlpackage.EscapeString(string(cl.fragment)))
```

Comment:
The line was quite long before already. What about putting the last parameter on a new line? I'd also rename the type `html` to `htmlPrinter` to avoid collision with the package's name.

Reply:
Done!